starter-core/src/main/java/io/micronaut/starter/feature/oraclecloud/OracleCloudAutonomousDatabase.java

```diff
@@ -100,6 +106,8 @@ public void apply(GeneratorContext generatorContext) {
             cfg.remove(jdbc.getUrlKey());
             cfg.remove(jdbc.getUsernameKey());
             cfg.remove(jdbc.getPasswordKey());
+            removeDatasourceDefaultConfig("dialect", "H2", cfg);
+            removeDatasourceDefaultConfig("schema-generate", "CREATE_DROP", cfg);
```

Comment:
Can you explain why this needs to be done?

Reply:
Since it is not in memory database I think the users do not expect the schema always recreated. Also I think we should kinda be consistent with the guideline for atp https://guides.micronaut.io/latest/micronaut-oracle-autonomous-db-gradle-java.html.